recipes/bleak/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "bleak" %}
+{% set version = "0.15.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bleak-{{ version }}.tar.gz
+  sha256: d8c8d88de0f22a15bd135ba056c4e5b2fb9f15119283def21b1ed7d43c00d590
+
+build:
+  entry_points:
+    - bleak-lescan=bleak:cli
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  # Mac Os X is currently not working as - pyobjc-framework-libdispatch is missing on conda-forge
```

Comment:
@ocefpaf I added the comment: 
```
# Mac Os X is currently not working as - pyobjc-framework-libdispatch is missing on conda-forge
```
And I am going to add the corresponding issue once the repository is created.